about-page.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Artificial intelligence forum</title>
+  </head>
+  <body>
+    <!-- popup menu area -->
+
+    <div id="close" class="js-menu hide margin-top show2 js-menua" >
+      <div class="menu-wrapper menu-wrappera">
+        <a id="cancel" class="hide hide2"
+          ><img class="canceli" src="IMG/Icon-Cancel.png" alt="icon"
+        /></a>
+        <a href="./index.html" class="log-nav-link"><img class="logo-nav log-nava" src="./IMG/itu-logo.png" alt="logo" /></a>
+        <ul class="js-list js-lista" id="mobile-menu-list">
+          <li>
+            <a href="./index.html" class="menu-list" id="elem1">Home</a>
+          </li>
+          <li>
+            <a href="#about" class="menu-list" id="elem2">UTI Projects</a>
+          </li>
+          <li>
+            <a href="#contact-me" class="menu-list nav-styled" id="elem3">Partners</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <!-- popup menu area -->
+
+    <header class="header-wrapper">
+        
+      <div class="menu-icon hide2" id="open"><i class="fa fa-reorder"></i></div>
+    </header>
+    <main>
+      <section class="about-section1">
+        <section class="bottom-section">
+          <h3 class="greetings">"Hello!smart world"</h3>
+          <h1 class="top-header">ITU EVENT</h1>
+        </section>
+        <section class="text-container text-containera">
+          <p class="text1 text1a">
+            This Forum was collocated with the ITU-T Study Group 20: Internet of
+            things (IoT) and smart cities and communities (SC&C) meeting which
+            took place from 3-13 December and the Joint Coordination Activity on
+            Internet of Things and Smart Cities and Communities (JCA-IoT and
+            SC&C) meeting which took place on 6 December 2018 in the same venue.
+          </p>
+        </section>
+        <section class="text2 contact-text-resize text2b">
+          <p>
+            Please contact us per Email for any further question about the ItU
+            Event, global forum on Artificial Intelligence and Internet of
+            things<br /><b><u>ituGlobalForum@itu.ca</u></b>
+          </p>
```

Comment:
- [ ] Kindly make sure to follow the background colour as given in the original design and also follow the spacing, font size for the webpage.

- Original design:
<img width="1265" alt="Screenshot 2022-04-15 at 3 33 33 PM" src="https://user-images.githubusercontent.com/70269507/163571291-c416936a-e2e2-4151-96c2-dcde1e24b09a.png">

- Your design:
<img width="736" alt="Screenshot 2022-04-15 at 3 33 53 PM" src="https://user-images.githubusercontent.com/70269507/163571300-49126659-1788-40c6-b7f3-bccb269b371f.png">